docs/CHANGELOG.txt

```diff
@@ -23,6 +23,72 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 
 Release Notes for PrestaShop 1.7
 
+####################################
+#   v1.7.8.8 - (2022-11-25)
```

Comment:
Maybe the date should be updated no? Ideally it should watch the release date, although it's a detail